.travis.yml

```diff
@@ -41,8 +41,7 @@ env:
 before_install:
   - docker run -d -p 5432:5432 --name augur_test_database augurlabs/augur:test_data@sha256:fd2d9a178a9fee7cd548bd40a16e08d4611be22892491e817aafd53502f74cd0
 install:
-  - ./scripts/install/backend.sh
-  - ./scripts/install/workers.sh
+  - pip install .[dev]
```

Comment:
You have an old version of this file in your branch. This change (getting rid of the shell scripts again) reverts a bug fix I made. If merged, this change will break the worker builds again. This change needs to be removed.